examples/js/README.md

```diff
@@ -28,33 +28,66 @@ grpc_tools_node_protoc --proto_path=../../protos/cardano-metadata-service/protob
 2. Set your username and API Token in the src/metadata_service_client.js file:
 ```shell
 const credentials = new CommonMessages.CredentialsMessage()
-    .setClientId("your_psg_username")
-    .setApiToken("your_api_token");
+    .setClientId("CLIENT_ID")
```

Comment:
Add in the comment that CLIENT_ID and API_TOKEN needs to be replaced (similarly to how it's done below)

Reply:
Fixed